src/WebUI/WorkLoads/OtherWorkloadsTable.tsx

```diff
@@ -168,21 +187,33 @@ export class OtherWorkloads extends BaseComponent<IOtherWorkloadsProperties, IOt
         return columns;
     }
 
-    private static _renderSetNameCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, statefulSet: ISetWorkloadTypeItem): JSX.Element {
-        return BaseKubeTable.renderTwoLineColumn(columnIndex, tableColumn, statefulSet.name, OtherWorkloads._getSetType(statefulSet.kind), css(colDataClassName, "two-lines", "zero-left-padding"), "primary-text", "secondary-text");
+    private static _renderSetNameCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, workload: ISetWorkloadTypeItem): JSX.Element {
+        return BaseKubeTable.renderTwoLineColumn(columnIndex, tableColumn, workload.name, OtherWorkloads._getSetType(workload.kind), css(colDataClassName, "two-lines", "zero-left-padding"), "primary-text", "secondary-text");
     }
 
-    private static _renderImageCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, statefulSet: ISetWorkloadTypeItem): JSX.Element {
-        const itemToRender = BaseKubeTable.renderColumn(statefulSet.image || "", BaseKubeTable.defaultColumnRenderer, colDataClassName);
+    private _renderImageCell = (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, workload: ISetWorkloadTypeItem): JSX.Element => {
+        const imageId = workload.imageId;
+        const imageText = workload.imageDisplayText;
+        // HardCoding hasImageDetails true for the time being, Should change it once we integrate with ImageService
+        //const hasImageDetails: boolean = this._hasImageDetails && this._hasImageDetails.hasOwnProperty(imageId) ? this._hasImageDetails[imageId] : false;
+        const hasImageDetails = true;
+        const itemToRender = hasImageDetails ?
```

Comment:
having code in two places is fine for the timebeing, if this has to be copied to more than see in future that we can have a common function, also see my code for the purpose of a link.

Reply:
Will revisit links padding once your PR is checked in